README.md

```diff
@@ -31,7 +31,7 @@ Add your answer to this markdown file. [This
 page](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/writing-mathematical-expressions)
 might help with the notation for mathematical expressions. 
 
-The solution is 3t(n/3)+O(n^5), I solved this by first looking at the recurive calls that are made which is three. Then looking at the nested loop
+The solution is 3t(n/3)+C(n^5) where C is a constant, I solved this by first looking at the recurive calls that are made which is three. Then looking at the nested loop
 it is n^2*n*n^2 which equals n^5.
 Using the backwards substituon method, first unfolding we get 9T(n/9)+O(n^5/3)+O(n^5) then we do a second unfolding we get 27T(n/27)+O(n^5/3^2)+O(n^5/3)+O(n^5), so the general form that we get is 3^kT(n/3^k)+O(n^5/(3^(k-1))+...+O(n^5/3)+O(n^5). If we simplify the gernal form 3^k=n => k= log3N, then we sub into summation, O 
```

Comment:
No, you can only use these time notations like big-oh when you conclude, to specify the complexity of that particular problem. Please look at the slides as a reference while writing backward substitution. At the end, please don't use equal sign like = O(n^5). It should be "\in" notation like what we use in set theory.